mlflow/mcp/server.py

```diff
@@ -0,0 +1,112 @@
+import contextlib
+import io
+from typing import Any, Callable, Literal
+
+import click
+from click import Parameter, ParamType
+from click.types import BOOL, FLOAT, INT, STRING, UUID
+from fastmcp import FastMCP
+from fastmcp.tools import FunctionTool
+
+
+@click.group()
+def cli():
+    """
+    Test CLI
+    """
```

Comment:
This is for quick testing. Will replace once https://github.com/mlflow/mlflow/pull/17302 lands.